CacheService/docker_context/CacheHandler.py

```diff
@@ -47,6 +48,19 @@ async def getUserFunds(self, user_id):
             else:
                 return result, status
 
+    async def getUser(self, user_id):
+        check = await self.RedisHandler.rExists(user_id)
+        if check:
+            data = await self.RedisHandler.rGet(user_id)
+            return data
+        else:
+            result, status = await self.RedisHandler.updateAccountCache(user_id)
+            if status == 200:
+                data = await self.RedisHandler.rGet(user_id)
+                return data
+            else:
+                return 404
```

Comment:
Why do we only return a data or result here and not both like the other methods?